nautobot/docs/development/best-practices.md

```diff
@@ -268,7 +268,8 @@ from nautobot.core.filters import NaturalKeyOrPKMultipleChoiceFilter
 
     provider = NaturalKeyOrPKMultipleChoiceFilter(
         queryset=Provider.objects.all(),
-        label="Provider (slug or ID)",
+        to_field_name="name",
+        label="Provider (name or ID)",
     )
 ```
 
```

Comment:
I'd expect to see a bunch of updates to `upgrading-from-nautobot-v1.md` as part of this PR?

Reply:
☹️